clang-tools-extra/docs/ReleaseNotes.rst

```diff
@@ -106,6 +106,9 @@ Improvements to clang-tidy
 - Improved :program:`run-clang-tidy.py` script. Fixed minor shutdown noise
   happening on certain platforms when interrupting the script.
 
+- Improved :program:`clang-tidy` by adding the option `CompilationArgsToRemoveRegex`
```

Comment:
Please move it after previous `clang-tidy` entry. Also ":option:`CompilationArgsToRemoveRegex`"?

Reply:
@EugeneZelenko, I might be wrong, but since the option is only mentioned in index.rst, I don't think I can use :option: in the Release notes.